servicemesh-install/topics/install.adoc

```diff
@@ -1,21 +1,65 @@
 [[installing-service-mesh]]
 = Installing the {ProductName}
 
-Installing the {ProductShortName} involves creating a custom resource definition file, then installing the operator to create and manage the custom resource.
+Installing the {ProductShortName} involves installing the operator, and then creating and managing a custom resource definition file to deploy the control plane.
 
 [NOTE]
 ====
-Starting with {ProductName} {ProductVersion}, Mixer’s policy enforcement is disabled by default, but you must enable it to run policy tasks. See https://docs.openshift.com/container-platform/3.11/servicemesh-install/servicemesh-install.html#update-mixer-policy-enforcement[Update Mixer policy enforcement] for instructions on enabling Mixer policy enforcement.
+Starting with {ProductName} 0.9.TechPreview, Mixer’s policy enforcement is disabled by default, but you must enable it to run policy tasks. See https://docs.openshift.com/container-platform/3.11/servicemesh-install/servicemesh-install.html#update-mixer-policy-enforcement[Update Mixer policy enforcement] for instructions on enabling Mixer policy enforcement.
```

Comment:
is disabled by default. You must enable it to run policy tasks.

Reply:
Fixed.

Reply:
Fixed.